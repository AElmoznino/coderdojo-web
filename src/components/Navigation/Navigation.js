import React from 'react'
import { Link } from 'react-router-dom'
import styled from 'styled-components'

import GridColumn from '../Grid/GridColumn'

const Wrap = styled(GridColumn)`
  background-color: ${({ theme }) => theme.colors.cornflowerBlue};
  padding-top: ${({ theme }) => theme.sizes.medium};
`

const List = styled.ul`
  display: flex;
  list-style-type: none;
  justify-content: space-between;
  height: ${({ theme }) => theme.sizes.large};
  left: 0;
  right: 0;
  top: 0;

  & a {
    color: ${({ theme }) => theme.colors.white};
    text-decoration: none;
  }

  & a:hover {
    font-weight: 700;
  }
`

export const Navigation = () => (
  <Wrap>
    <List>
      <li>
        <Link to="/">Start</Link>
      </li>
      <li>
        <Link to="/about">Om</Link>
      </li>
    </List>
  </Wrap>
)

export default Navigation
