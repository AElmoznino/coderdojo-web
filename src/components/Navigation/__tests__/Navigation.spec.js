import React from 'react'
import { shallow } from 'enzyme'
import Navigation from '../Navigation'

describe('components/Navigation', () => {
  let component

  beforeEach(() => {
    component = shallow(<Navigation />)
  })

  it('renders Navigation', () => {
    expect(component).toMatchSnapshot()
  })
})
