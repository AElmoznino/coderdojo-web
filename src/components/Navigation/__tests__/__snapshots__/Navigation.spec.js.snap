// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Navigation renders Navigation 1`] = `
<Styled(styled.div)>
  <styled.ul>
    <li>
      <Link
        replace={false}
        to="/"
      >
        Start
      </Link>
    </li>
    <li>
      <Link
        replace={false}
        to="/about"
      >
        Om
      </Link>
    </li>
  </styled.ul>
</Styled(styled.div)>
`;
